nodemon.json

```
@@ -0,0 +1,6 @@
+{
+    "watch": ["public", "locales", "components", "build.js"],
+    "exec": "npm run build && static-server -p 8080 dist",
```

```suggestion
    "exec": "npm run build && static-server -p 8058 dist",
```

I'm developing many apps and often causes port collision so I want to use other port 8058